pallets/passkey/src/mock.rs

```diff
@@ -74,9 +75,35 @@ impl pallet_balances::Config for Test {
 	type RuntimeHoldReason = ();
 }
 
+pub struct MockPasskeyCallFilter;
+
+impl Contains<RuntimeCall> for MockPasskeyCallFilter {
+	fn contains(call: &RuntimeCall) -> bool {
+		match call {
+			RuntimeCall::System(frame_system::Call::remark { .. }) |
```

Comment:
I was not expecting to see remark here. I am curious to know why we included it?

Reply:
It's not easy creating non frame calls (like Balance::transfer) from inside benchmarks and that's why we are using only remark inside benchmarks. When we are running benchmarks as test we are consuming MockPasskeyCallFilter which means we need to have remark also as part of the valid calls.

Reply:
Sounds reasonable. It seems that we might benefit from a feature flag instead?

Reply:
I had it featured flagged first but removed it to simplify. I can add it again if you think that would be better.

Reply:
I believe it might be better to use a feature flag since we added this primarily to make benchmarking and testing easier. As far as I know, we only intend to allow users to transfer balances for now.

Reply:
Clsoing this since I added it as part of AC for https://github.com/frequency-chain/frequency/issues/2032